plugins/modules/ini_file.py

```diff
@@ -46,6 +45,21 @@
       - If left empty, being omitted, or being set to V(null), the O(option) will be placed before the first O(section).
       - Using V(null) is also required if the config format does not support sections.
     type: str
+  section_has:
+    type: list
+    elements: dict
+    required: false
+    suboptions:
+      option:
+        type: str
+        description: the option to look for within the section
```

Comment:
Paragraphs should always consist of proper sentences.
```suggestion
        description: The option to look for within the section.
```